src/analysis/calculus/fderiv.lean

```diff
@@ -2095,22 +2095,22 @@ lemma fderiv_const_mul (hc : differentiable_at 𝕜 c x) (d : 𝕜) :
 end mul
 
 section algebra_inverse
-variables {R :Type*} [normed_ring R] [normed_algebra 𝕜 R] [complete_space R]
+variables {𝕜' :Type*} [normed_ring 𝕜'] [normed_algebra 𝕜 𝕜'] [complete_space 𝕜']
```

Comment:
This shouldn't be `𝕜'` since it's not a field, `R` was fine...

Reply:
Fixed, thanks.  I had picked up the notation from `normed_space.basic`, for example 
```
class normed_algebra (𝕜 : Type*) (𝕜' : Type*) [normed_field 𝕜] [normed_ring 𝕜'] extends algebra 𝕜 𝕜'
```
is that notation an older style?

Reply:
The general rule is `R` for rings and `𝕜` for fields. In the line you're citing, `𝕜'` should also be `R`, yes.